src/App.tsx

```diff
@@ -9,6 +11,9 @@ import {Calculator} from './components/pages/Calculator';
 import {Homepage} from './components/pages/Homepage';
 import {SignIn} from './components/pages/SignIn';
 import {SignUp} from './components/pages/SignUp';
+import {PrivateRoute} from './components/auth/PrivateRoute';
+import {PublicRoute} from './components/auth/PublicRoute';
+
```

Comment:
A tip I like to do is comment sections of imports so everyone knows where to add theirs. so components should be grouped together under a components comment etc...etc...

Reply:
Yes sir! I'll find and add an eslint rule to make sure that this is always enforced.

Reply:
I added an eslint rule to group imports, but commenting all files would have been a hassle, so I left that part out.